feature/gribi/basic_encap/README.md

```diff
@@ -0,0 +1,389 @@
+# TE-16.1: basic encapsulation tests
```

Comment:
can we move the README.md file for TE-16.1 to a folder like feature/gribi/otg_tests/basic_encap_test ? or any suitable name. I have raised a PR with implementation of this test using PR https://github.com/openconfig/featureprofiles/pull/2456
But rundata test will fail in the absence of README.md file.

Reply:
feel free to move this folder along as part of your PR. I"m totally ok with that.